app/src/main/java/com/kickstarter/ui/adapters/ProjectAdapter.java

```diff
@@ -33,33 +26,23 @@ public ProjectAdapter(final @NonNull Delegate delegate) {
     if (sectionRow.section() == 0) {
       return R.layout.project_main_layout;
     } else {
-      return R.layout.reward_view;
+      return R.layout.empty_view;
     }
   }
 
   /**
    * Populate adapter data when we know we're working with a ProjectData object.
    */
-  public void takeProject(final @NonNull ProjectData projectData, final @NonNull Boolean nativeCheckoutEnabled) {
+  public void takeProject(final @NonNull ProjectData projectData) {
```

Comment:
Just for clarification, these rewards are now moved into the native checkout flow, which is what allows us to delete all of this code?

Reply:
Yep! And I plan to open another PR to rename `NativeCheckoutRewardViewHolder` to `RewardViewHolder`